chromadb/utils/embedding_functions/jina_embedding_function.py

```diff
@@ -25,6 +31,19 @@ def __init__(
                 Defaults to "CHROMA_JINA_API_KEY".
             model_name (str, optional): The name of the model to use for text embeddings.
                 Defaults to "jina-embeddings-v2-base-en".
+            task (str, optional): The task to use for the Jina AI API.
+                Defaults to None.
```

Comment:
do you think it make sense to change the default model from `jina-embeddings-v2-base-en` to `jina-embeddings-v3`? will it break ppl's user experience? If so the default task would be nice to set to `text-matching`